supplemental_notebooks/2020-07-30_pathway_analysis.Rmd

```diff
@@ -617,86 +636,102 @@ fit2 <- eBayes(fit)
 ```
 
 Let's take a look at the top 10 pathways returned.
-(To look just at the pathways for the Group 3 digoxin vs. no treatment, we could do `topTable(fit2, coef = 1)`, instead.)
-
-```{r}
-topTable(fit2, number = 10)
-```
-
-It can be helpful to visualize these pathways individually, with something like a jitter plot rather than a heatmap.
-
-```{r}
-data.frame(gsva_score = gsva_results["HALLMARK_TGF_BETA_SIGNALING", ],
-           subtype_treatment) %>%
-  ggplot2::ggplot(ggplot2::aes(x = subtype_treatment,
-                               y = gsva_score)) +
-  ggplot2::geom_jitter(width = 0.1, alpha = 0.5) +
-  ggplot2::theme_bw() +
-  ggplot2::stat_summary(
-    geom = "point",
-    fun = "mean",
-    size = 3,
-    color = "blue",
-    pch = 18
-  ) +
-  ggplot2::ggtitle("HALLMARK_TGF_BETA_SIGNALING")
-```
-
-```{r}
-data.frame(gsva_score = gsva_results["HALLMARK_INTERFERON_GAMMA_RESPONSE", ],
-           subtype_treatment) %>%
-  ggplot2::ggplot(ggplot2::aes(x = subtype_treatment,
-                               y = gsva_score)) +
-  ggplot2::geom_jitter(width = 0.1, alpha = 0.5) +
-  ggplot2::theme_bw() +
-  ggplot2::stat_summary(
-    geom = "point",
-    fun = "mean",
-    size = 3,
-    color = "blue",
-    pch = 18
-  ) +
-  ggplot2::ggtitle("HALLMARK_INTERFERON_GAMMA_RESPONSE")
-```
-
-```{r}
-data.frame(gsva_score = gsva_results["HALLMARK_G2M_CHECKPOINT", ],
-           subtype_treatment) %>%
-  ggplot2::ggplot(ggplot2::aes(x = subtype_treatment,
-                               y = gsva_score)) +
-  ggplot2::geom_jitter(width = 0.1, alpha = 0.5) +
-  ggplot2::theme_bw() +
-  ggplot2::stat_summary(
-    geom = "point",
-    fun = "mean",
-    size = 3,
-    color = "blue",
-    pch = 18
-  ) +
-  ggplot2::ggtitle("HALLMARK_G2M_CHECKPOINT")
-```
-
-```{r}
-data.frame(gsva_score = gsva_results["HALLMARK_HEDGEHOG_SIGNALING", ],
-           subtype_treatment) %>%
-  ggplot2::ggplot(ggplot2::aes(x = subtype_treatment,
-                               y = gsva_score)) +
-  ggplot2::geom_jitter(width = 0.1, alpha = 0.5) +
-  ggplot2::theme_bw() +
-  ggplot2::stat_summary(
-    geom = "point",
-    fun = "mean",
-    size = 3,
-    color = "blue",
-    pch = 18
-  ) +
-  ggplot2::ggtitle("HALLMARK_HEDGEHOG_SIGNALING")
+Without specifying a coefficient, `topTable()` will report the F-tests statistics for each of the three contrasts: treatment in Group 3, treatment in Group 4, and differences in response between the subtypes.
```

Comment:
It looked to me like it was doing something more like an F test (ANOVA) for all groups; i.e. not looking at specific contrasts at all? I guess the test is if you only specified 1 contrast, would the results for this table be any different?

Reply:
> I guess the test is if you only specified 1 contrast, would the results for this table be any different?

Do you mean use `coef` or something else?

Reply:
I mean if the `contrast_matrix` was made with only one contrast, would `topTable()` still give the same F statstistics here?